ui/Screen/Project/Source.svelte

```diff
@@ -100,6 +102,7 @@
     selectRevision(revision);
   };
 
+  onDestroy(watchPatchUpdates());
```

Comment:
Should this not be registered in `onMount`? And `onDestroy` have the event listener stop listening?

Reply:
> Should this not be registered in `onMount`? And `onDestroy` have the event listener stop listening?

This is what the code does. When it is executed (i.e. on mount) `watchPatchUpdates()` is called. The function it returns is then registered as a callback on destroy.

Reply:
Hm, got it. Is it possible to make this a bit more explicit? From looking at the code it's not very obvious.

Reply:
Yes, I think it makes sense to be more explicit here: https://github.com/radicle-dev/radicle-upstream/pull/1821/commits/6d7859f1a1f626a2d89f5e272e15e7073f9f017b